assignment-2/src/view/DumbComponents/QuestionsList.js

```diff
@@ -0,0 +1,131 @@
+// DUMB Component
+import React from 'react';
+import '../../App.css';
+import {UncontrolledCollapse, CardBody, Card} from 'reactstrap';
+import MyNavbar from './MyNavbar';
+import {FaChevronDown, FaChevronUp} from "react-icons/fa";
+
+
+const QuestionsList = ({
```

Comment:
This is a huge component --> split it into multiple smaller ones. For example the cards inside the map calls could be dumb components as well.